migrations/1599673919062-lernstore_permission_SH.js

```diff
@@ -0,0 +1,218 @@
+const mongoose = require('mongoose');
+const { connect, close } = require('../src/utils/database');
+const logger = require('../src/logger');
+
+const FEDERALSTATE_ABBR = 'SH';
```

Comment:
why are these constants in uppercase?

Reply:
it is not unusual that constants which have only 1 static value (i.e. not a function) are written in uppercase. We have this in many existing scripts already